src/primo/browzine-primo-adapter.js

```diff
@@ -248,6 +248,7 @@ browzine.primo = (function() {
     if (!isArticle(scope)) {
       return false;
     }
+    //We can maybe get rid of this now? Per Karl, the reasons for doing this may no longer matter. May need more investigation.
```

Comment:
I think it's fine to keep this and eliminate the comment, since this will keep on working fine without any change.  But if we want to eliminate it, sure, we can axe it and see how it behaves.  But I think in this case, since we don't change this too often, and don't mess with unpaywall too much, we can probably just keep it the same. This is a pretty defensive approach to this code, since it checks to see if the API response even has this element, and if it doesn't does the right thing.  If we ever get rid of that element, it would be nice to know we have defensive code deployed that likely won't break.